src/test/java/org/folio/rest/api/CallNumberNormalizedTestBase.java

```diff
@@ -0,0 +1,151 @@
+package org.folio.rest.api;
+
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+import java.util.List;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+
+@RunWith(JUnitParamsRunner.class)
+public abstract class CallNumberNormalizedTestBase extends TestBaseWithInventoryUtil {
+
+  protected abstract List<String> searchByCallNumberNormalized(String callNumber) throws Exception;
+
+  @Parameters({
+    "germ 350",
+    "germ350",
+    "Germ 350/"
+  })
+  @Test
+  public void appliesRightTruncationToSearchQuery(String searchQuery) throws Exception {
+    final List<String> callNumbersFound = searchByCallNumberNormalized(searchQuery);
+
+    assertThat(callNumbersFound, hasSize(3));
+    assertThat(callNumbersFound, allOf(
+      hasItem("H Germ 350/35: 1"),
+      hasItem("H Germ 350/35: 2"),
+      hasItem("F Germ 350/1")
+    ));
+  }
+
+  @Test
+  public void canApplyRightTruncationForSuffix() throws Exception {
```

Comment:
Does this test demonstrate that the search term can also find matches when it includes part of the suffix?

```suggestion
  public void canSearchByCallNumberAndPartOfSuffix() throws Exception {
```

Reply:
Yes, the test demonstrates that if user entered call number + part of the suffix, then right truncation for the suffix should be applied.

Reply:
Ok, thanks, I wanted to confirm my assumption to go with my suggestion